Umbraco-Uno/Creating-Content/How-to-Set-up-a-Blog with-a-Feed/index.md

```diff
@@ -0,0 +1,51 @@
+
+---
+versionFrom: 8.0.0
+---
+
+# The Blog
+In this guide you will learn, to set up a blog with a feed that contains the blog posts. 
+
+When you are done with this guide you will have a blog that looks something like this:
+
+![final blog](images/Final-blog.png)
+
+## How to set up the blog
+
+We will start by making the blog page, which will contain the posts we will create in the feed.
+
+1. First, navigate to the content tree on the left
+2. In the content tree hover over Unicorn Hotel and click ... button
+3. After clicking the ... button select feed from the menu that shows up
+4. Name this what you would like it to be called in this case we will name it Blog(if there's already a blog page you might have to delete that first)
+5. After naming the blog page, go to widgets and select the hero widget
+6. Set up the hero widget as you would like it and hit ***save*** or ***save and publish***
+7. After doing this the Blog page itself is done
+
+## How to set up the feed
+
+In this part, we will set up the feed for our blog, so that we will have some posts on our blog page.
+
+1. Click the ... button next to the Blog node in the tree
+2. Select Post
+3. Give the post a name
+4. Fill out the content section with what you want the blog post to be about
+5. Head to the Meta section
+6. In the date section select the publishing date for the post
+7. The image here is a thumbnail for the post
+8. Finalize by writing a short description of the post and hit ***save** or ***save and publish***
+
+## How to add more blog posts to the feed
+
+A blog usually consists of multiple posts, so this is how you add more blog posts after you have created the first blog post.
```

Comment:
A blog usually consists of multiple posts, so add however many blog pages you like by following the steps below.

Reply:
A blog usually consists of multiple posts, so you can add however many blog posts you like by repeating the steps below.